src/input_parsers/PropertyParser.h

```diff
@@ -17,31 +17,28 @@
 #define __PropertyParser_h__
 
 #include "Equation.h"
-#include "InputQuery.h"
+#include "IQuery.h"
 #include "MString.h"
+#include "Query.h"
 
 /*
   This class reads a property from a text file, and stores the property's
-  constraints within an InputQuery object.
+  constraints within an Query object.
```

Comment:
But the signature takes an `IQuery`, not a `Query`?

Reply:
Updated documentation to IQuery